[2024-07-03T09:57:46.644+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T11:40:00+00:00 [queued]>
[2024-07-03T09:57:46.653+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T11:40:00+00:00 [queued]>
[2024-07-03T09:57:46.653+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T09:57:46.676+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 11:40:00+00:00
[2024-07-03T09:57:46.687+0000] {standard_task_runner.py:57} INFO - Started process 20612 to run task
[2024-07-03T09:57:46.703+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T11:40:00+00:00', '--job-id', '385', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmp8aww5sua']
[2024-07-03T09:57:46.704+0000] {standard_task_runner.py:85} INFO - Job 385: Subtask id_task_get_emails
[2024-07-03T09:57:46.791+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T11:40:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T09:57:46.930+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T11:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T11:40:00+00:00'
[2024-07-03T09:57:46.931+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T09:57:46.931+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_email_robot.py", line 22, in get_emails
    email_conf = json.load (open(path / ("config/email_conf.json")))
NameError: name 'json' is not defined
[2024-07-03T09:57:46.945+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T114000, start_date=20240703T095746, end_date=20240703T095746
[2024-07-03T09:57:46.968+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 385 for task id_task_get_emails (name 'json' is not defined; 20612)
[2024-07-03T09:57:46.995+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-07-03T09:57:47.015+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T12:05:16.091+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T11:40:00+00:00 [queued]>
[2024-07-03T12:05:16.127+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T11:40:00+00:00 [queued]>
[2024-07-03T12:05:16.127+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T12:05:16.176+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 11:40:00+00:00
[2024-07-03T12:05:16.186+0000] {standard_task_runner.py:57} INFO - Started process 23432 to run task
[2024-07-03T12:05:16.195+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T11:40:00+00:00', '--job-id', '826', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmpn2b2sfqp']
[2024-07-03T12:05:16.196+0000] {standard_task_runner.py:85} INFO - Job 826: Subtask id_task_get_emails
[2024-07-03T12:05:16.287+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T11:40:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T12:05:16.413+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T11:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T11:40:00+00:00'
[2024-07-03T12:05:16.414+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T12:05:16.417+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:05:16.417398_User@StatDep: EmailRobot in screated.
[2024-07-03T12:05:16.417+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:05:16.417607_User@StatDep: get_for_recent_emails is started...
[2024-07-03T12:05:16.417+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:05:16.417744_User@StatDep: pop3_fetch_emails isstarted...
[2024-07-03T12:05:16.828+0000] {logging_mixin.py:151} INFO -    b'+OK'
[2024-07-03T12:05:17.305+0000] {logging_mixin.py:151} INFO -    mailbox have 1 emails and the robot can process 10
[2024-07-03T12:05:17.732+0000] {logging_mixin.py:151} INFO -    #>2024-07-03 12:05:17.732521_conneciton to db is set.
[2024-07-03T12:05:17.742+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_email_robot.py", line 25, in get_emails
    robot.fetch_emails()
  File "/opt/airflow/dags/emailRobot.py", line 62, in fetch_emails
    return self.pop3_fetch_emails()
  File "/opt/airflow/dags/emailRobot.py", line 156, in pop3_fetch_emails
    cursor.execute (query, insert_values)
psycopg2.errors.UndefinedTable: relation "sma_stat_dep.tbl_file_upload" does not exist
LINE 1: ...) AS (INSERT INTO                                 sma_stat_d...
                                                             ^

[2024-07-03T12:05:17.761+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T114000, start_date=20240703T120516, end_date=20240703T120517
[2024-07-03T12:05:17.796+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 826 for task id_task_get_emails (relation "sma_stat_dep.tbl_file_upload" does not exist
LINE 1: ...) AS (INSERT INTO                                 sma_stat_d...
                                                             ^
; 23432)
[2024-07-03T12:05:17.838+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-07-03T12:05:17.875+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T12:24:15.976+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T11:40:00+00:00 [queued]>
[2024-07-03T12:24:16.007+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T11:40:00+00:00 [queued]>
[2024-07-03T12:24:16.007+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T12:24:16.140+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 11:40:00+00:00
[2024-07-03T12:24:16.145+0000] {standard_task_runner.py:57} INFO - Started process 24393 to run task
[2024-07-03T12:24:16.156+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T11:40:00+00:00', '--job-id', '1161', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmpyb76kshm']
[2024-07-03T12:24:16.158+0000] {standard_task_runner.py:85} INFO - Job 1161: Subtask id_task_get_emails
[2024-07-03T12:24:16.231+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T11:40:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T12:24:16.311+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T11:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T11:40:00+00:00'
[2024-07-03T12:24:16.312+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T12:24:16.314+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:24:16.314605_User@StatDep: EmailRobot in screated.
[2024-07-03T12:24:16.314+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:24:16.314800_User@StatDep: get_for_recent_emails is started...
[2024-07-03T12:24:16.314+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:24:16.314927_User@StatDep: pop3_fetch_emails isstarted...
[2024-07-03T12:24:16.701+0000] {logging_mixin.py:151} INFO -    b'+OK'
[2024-07-03T12:24:17.001+0000] {logging_mixin.py:151} INFO -    mailbox have 0 emails and the robot can process 10
[2024-07-03T12:24:17.002+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:24:17.002429_User@StatDep: no emails to process, exiting pop3_fetch_emails()...
[2024-07-03T12:24:17.003+0000] {python.py:194} INFO - Done. Returned value was: True
[2024-07-03T12:24:17.043+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T114000, start_date=20240703T122415, end_date=20240703T122417
[2024-07-03T12:24:17.094+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-07-03T12:24:17.123+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
